spec/index.html

```diff
@@ -1753,6 +1758,16 @@ <h3>RDF Term Constructors</h3>
       <tr id="handle-blankNodePropertyList"           ><td style="text-align:left;"            ><a class="type bNode"        href="#grammar-production-blankNodePropertyList"           >blankNodePropertyList           </a></td><td><a data-cite="RDF12-CONCEPTS#dfn-blank-node">   blank node   </a></td><td>A blank node is generated. Note the rules for <code>blankNodePropertyList</code> in the next section.</td></tr>
       <tr id="handle-collection"                      ><td style="text-align:left;" rowspan="2"><a class="type bNode"        href="#grammar-production-collection"                      >collection                      </a></td><td><a data-cite="RDF12-CONCEPTS#dfn-blank-node">   blank node   </a></td><td>For non-empty lists, a blank node is generated. Note the rules for <code>collection</code> in the next section.</td></tr>
       <tr id="handle-collection2"                     >                                                                                                                                                                           <td><a data-cite="RDF12-CONCEPTS#dfn-iri">          IRI          </a></td><td>For empty lists, the resulting IRI is <code>rdf:nil</code>. Note the rules for <code>collection</code> in the next section.</td></tr>
+      <tr id="handle-reifier"                         ><td style="text-align:left;"            ><a class="type reifier" href="#grammar-production-reifier"                              >reifier                         </a></td><td><a data-cite="RDF12-CONCEPTS#dfn-iri">          IRI          </a>
+                                                                                                                                                                                                                                    | <a data-cite="RDF12-CONCEPTS#dfn-blank-node">   blank node   </a></td>
+        <td>
+          The <a data-cite="RDF12-CONCEPTS#dfn-rdf-term">term</a> sets |curReifier| and is taken from the matched
+          <a href="#grammar-production-iri"><code>iri</code></a> production
+          or <a href="#grammar-production-BlankNode"><code>BlankNode</code></a> production, if any.
+          If no such production is matched, it is taken
+          from a fresh RDF <a data-cite="RDF12-CONCEPTS#dfn-blank-node">blank node</a>.
```

Comment:
"The `term` sets `curReifier` [...]." What's "taken from the matched ... production"? Is that `term` or `curReifier`? I'm guessing here, so this may be wrong.
```suggestion
          The <a data-cite="RDF12-CONCEPTS#dfn-rdf-term">term</a> sets |curReifier| which is taken from the matched
          <a href="#grammar-production-iri"><code>iri</code></a> production
          or <a href="#grammar-production-BlankNode"><code>BlankNode</code></a> production, if any.
          If no such production is matched, |curReifier| is taken
          from a fresh RDF <a data-cite="RDF12-CONCEPTS#dfn-blank-node">blank node</a>.
```

Reply:
This might be more correct?
```suggestion
          The |curReifier| is set from <a data-cite="RDF12-CONCEPTS#dfn-rdf-term">term</a>, which is taken from the matched
          <a href="#grammar-production-iri"><code>iri</code></a> production
          or <a href="#grammar-production-BlankNode"><code>BlankNode</code></a> production, if any.
          If no such production is matched, <a data-cite="RDF12-CONCEPTS#dfn-rdf-term">term</a> is taken
          from a fresh RDF <a data-cite="RDF12-CONCEPTS#dfn-blank-node">blank node</a>.
```